core/templates/dev/head/services/exploration-features.service.ts

```diff
@@ -16,7 +16,18 @@
  * @fileoverview Service for determining the visibility of advanced features in
  *               the exploration editor.
  */
-
+export interface ExplorationDataInterface {
+  'param_changes': string[],
+  'states': {
+    [propsName : string]: {
+      'param_changes': string[]
+    }
+  }
+}
+export interface FeatureDataInterface {
```

Comment:
Please fix the name as ```IFeatureDataInterface```

Reply:
i have resolved all the changes in a commit
thanks